### PR TITLE
SEO improvements for learn.microsoft

### PR DESCRIPTION
**Pull request by joeyobr**
John, pls review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/aafa78de09bf7eee63a1e844a68661ff0b69ba50/globalization%2Findex.yml) | [100](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8a233d7-1388-4e41-b55d-05fbdc09f424) | 100 | 100 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/aafa78de09bf7eee63a1e844a68661ff0b69ba50/globalization%2Finput%2Fjapanese-ime.md) | [92](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/70768049-1dee-4f71-af13-1f57780a6a95) | 100 | 83 | 79 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/f99a2a4f52f2a3e9e2f039683306dd069ef9d0ce/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3da05a01-ab61-4e2d-8592-f806e19c7203) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/f99a2a4f52f2a3e9e2f039683306dd069ef9d0ce/globalization%2Finput%2Fjapanese-ime.md) | [92](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/20f763cc-6e80-4139-afbe-1a5989b0e122) | 100 | 83 | 79 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/2508e51a27a735983033edd43f6d115c7c4b3ea9/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b32077f2-2e54-49e0-ad11-20db8047b1ea) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/2508e51a27a735983033edd43f6d115c7c4b3ea9/globalization%2Finput%2Fjapanese-ime.md) | [93](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f5cf9d0-24a4-41c7-9300-e12d24437b83) | 100 | 94 | 79 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5c7d5ff0-c53f-4ae0-be19-4fd57a719195) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Finput%2Fjapanese-ime.md) | [93](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87cbd8b4-318e-4c4a-88bf-cafd6cc735f5) | 100 | 94 | 79 |
| [:white_check_mark:](# "Successfully processed") [globalization/localization/localization-overview.md](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Flocalization%2Flocalization-overview.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93ea60e6-baf7-45e5-a1a5-5a9481cf0974) | 100 | 91 | 48 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [85](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/95753329-815a-45cb-93d1-fb1d8c33efbe) | 100 | 85 | 73 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/international-design.md](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Fmethodology%2Finternational-design.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/04150d7a-a9eb-400b-8bb3-cf9c4caec48f) | 100 | 65 | 59 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/software-internationalization.md](https://github.com/MicrosoftDocs/globalization/blob/66ba0d1da4ec7670f6465099d674a6554b8b023d/globalization%2Fmethodology%2Fsoftware-internationalization.md) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dfeb45bd-2346-4545-aceb-a163742f8922) | 100 | 93 | 77 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/index.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ### YamlMime:Landing
 
 title: Globalization documentation
-summary: With growing software demands all over the world, you should design and implement software to support various languages and data formats.
+summary: Explore resources for designing and implementing software to support various languages and data formats.
 metadata:
-  title: Software internationalization documentation 
-  description: Explore resources for internationalization (globalization + localizability + localization).
+  title: Software globalization documentation 
+  description: Explore resources for globalization (internationalization + localization)/(i18n + l10n).
```
Comment (jowilco) [inline globalization/index.yml:7]: How about, "Explore resources for globalization (g11n), internationalization (i18n), and localization (l10n)."

File: globalization/input/japanese-ime.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ To open the IME toolbar, open the right-click menu and select **IME toolbar**. I
 
 ## Example
 
-The Japanese word for summer is 夏, which is a single kanji character. Its phonetic romanization is "natsu."
+The Japanese word for summer is 夏, which is a single kanji character. Its phonetic romanization is "natsu".
```
Comment (jowilco) [inline globalization/input/japanese-ime.md:113]: https://learn.microsoft.com/en-us/style-guide/punctuation/quotation-marks

File: globalization/localization/localization-overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Localizing your product
-description: This article helps you understand the process of localization and strategies for localizing different types of content.
+title: Localize your product
+description: Understand the process of localization and strategies for localizing different types of content. (localize/l10n)
```
Comment (jowilco) [inline globalization/localization/localization-overview.md:3]: Understand the process of localization (l10n) and strategies for how to localize different types of content.

File: globalization/methodology/global-product-delivery.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Global product delivery
-description: A great global product has a rich experience that feels like it was created specifically for you, no matter where you live or the language you speak.
+description: Deliver a great product experience for all customers, no matter where they live or what language they speak. (global delivery)
```
Comment (jowilco) [inline globalization/methodology/global-product-delivery.md:3]: I don't think that we need to include " (global delivery)" as this isn't a key term for us.

File: globalization/methodology/international-design.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
 ---
-title: International design
-description: It's important to think about internationalization from the very beginning of a software project. This article provides an extensive checklist of internationalization requirements when developing software.
+title: Design software for international markets
+description: Understand how to design software for international markets. (internationalization/i18n/world readiness)
```
Comment (jowilco) [inline globalization/methodology/international-design.md:3]: Understand how to design software for international markets by using internationalization (i18n) best practices.

File: globalization/methodology/software-internationalization.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Software Internationalization
-description: Design and develop your application for international markets from the beginning to ensure global support.
+description: Design and develop your application for international markets from the beginning to ensure global support. (Internationalization/i18n)
```
Comment (jowilco) [inline globalization/methodology/software-internationalization.md:3]: Design and develop your application using internationalization (i18n) best practices to ensure global support.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c28ccfba-1fc6-4f05-9ece-db82fc19adaf) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Finput%2Fjapanese-ime.md) | [93](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7143ef16-2df4-425e-be4d-05651855ac55) | 100 | 94 | 79 |
| [:white_check_mark:](# "Successfully processed") [globalization/localization/localization-overview.md](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Flocalization%2Flocalization-overview.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e4c239c-4e1a-444f-8687-3623b68f1e62) | 100 | 88 | 48 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [85](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4cc11ef8-0b20-46c0-9177-bbdeed918978) | 100 | 85 | 73 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/international-design.md](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Fmethodology%2Finternational-design.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/684c32c4-0eb0-4de5-a6bd-00828b2e89e8) | 100 | 65 | 59 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/software-internationalization.md](https://github.com/MicrosoftDocs/globalization/blob/45a267fa752e3f4722178a12b69aab8030d3ae9d/globalization%2Fmethodology%2Fsoftware-internationalization.md) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64d14d26-5778-4c36-9b04-7efe89f4b601) | 100 | 93 | 77 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22f86404-a85c-464c-8fad-9e3bb3ee85f2) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Finput%2Fjapanese-ime.md) | [93](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c895f6f-30f4-48ec-9fe7-0a94874828b4) | 100 | 83 | 83 |
| [:white_check_mark:](# "Successfully processed") [globalization/localization/localization-overview.md](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Flocalization%2Flocalization-overview.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/44e9c7ad-f711-49f9-8279-f41b08d41797) | 100 | 88 | 48 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [87](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9771e478-c7c6-4b20-85a1-ee9eb95d061e) | 100 | 92 | 73 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/international-design.md](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Fmethodology%2Finternational-design.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c6ffbf1-ac7b-4c5d-8298-96fbfd1cd6c1) | 100 | 65 | 59 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/software-internationalization.md](https://github.com/MicrosoftDocs/globalization/blob/49071ba000689a61550046244d7cac40bc158979/globalization%2Fmethodology%2Fsoftware-internationalization.md) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f465fc84-b2d1-42c3-87af-da5e93c1dfe8) | 100 | 93 | 77 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/index.yml](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Findex.yml) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/47ab7131-82af-4ff3-987e-57c98237e187) | 100 | 70 | 100 |
| [:white_check_mark:](# "Successfully processed") [globalization/input/japanese-ime.md](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Finput%2Fjapanese-ime.md) | [93](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2ed21ec9-a1e6-4bde-9878-1798490e0e03) | 100 | 83 | 83 |
| [:white_check_mark:](# "Successfully processed") [globalization/localization/localization-overview.md](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Flocalization%2Flocalization-overview.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/375149ae-b99e-407d-8d06-1cde3a72fd36) | 100 | 88 | 48 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [87](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da137a00-3b13-4ba1-8ea5-53e4c6ae1970) | 100 | 92 | 73 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/international-design.md](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Fmethodology%2Finternational-design.md) | [80](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/15b2dbfe-2be6-4c2f-8a7a-7e54e7433743) | 100 | 65 | 60 |
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/software-internationalization.md](https://github.com/MicrosoftDocs/globalization/blob/04e5c48c042ee9ad79a712081040f33d03307257/globalization%2Fmethodology%2Fsoftware-internationalization.md) | [94](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/81f5363b-6ded-46d4-a3a4-f59e2c7829ca) | 100 | 93 | 77 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by jowilco — APPROVED** [review]

**Pull request merged**